docs/building-modules/upgrade.md

```diff
@@ -0,0 +1,42 @@
+<!--
+order: 13
+-->
+
+# Upgrade
+
+The Upgrade module allows your modules to smoothly transition to new versions with breaking changes. This document outlines how to build modules to take advantage of this functionality. 
+
```

Comment:
```suggestion
## Pre-requisite Readings

- [In-Place Store Migration](../core-concepts/upgrade.md) {prereq}
```

What do you think about adding this section? `{prereq}` renders in a special way in vuepress, e.g.  http://docs.cosmos.network/v0.42/building-modules/query-services.html on the right side.